sql/core/src/test/resources/sql-tests/inputs/udf/udf-inline-table.sql

```diff
@@ -0,0 +1,54 @@
+-- This test file was converted from intersect-all.sql.
+-- Note that currently registered UDF returns a string. So there are some differences, for instance
+-- in string cast within UDF in Scala and Python.
+
+-- single row, without table and column alias
+select * from values ("one", 1);
+
+-- single row, without column alias
+select * from values ("one", 1) as data;
```

Comment:
let's explicitly test UDF here instead of `*`.

Reply:
Done.